### PR TITLE
Add Bifrost testnet Asgard to testnet list

### PR DESCRIPTION
**Pull request by janpo**

### CODE REVIEW

File: packages/apps-config/src/ui/logos/index.ts [hunk 1]
```diff
@@ -78,6 +79,8 @@ const chainLogos: Record<string, any> = [
 // alphabetical
 const nodeLogos: Record<string, any> = [
   ['airalab-robonomics', nodeRobonomics],
+  ['Bifrost Node', nodeBifrost],
+  ['Bifrost', nodeStafi],
```
Comment (jacogr) [inline packages/apps-config/src/ui/logos/index.ts:83]:
```suggestion
  ['Bifrost', nodeStafi],
```

I believe this is incorrect?

**Review by jacogr — COMMENTED** [review]
One small comment, apart from that it looks sane.

File: packages/apps-config/src/ui/logos/index.ts (hunk 1 above)
Comment (janpo) [reply packages/apps-config/src/ui/logos/index.ts:83]: @jacogr  Thank you very much.

**Review by jacogr — APPROVED** [review]
Thank you! (Will merge in the next short while)

**Pull request merged**